src/frontend/components/AppComponent.tsx

```diff
@@ -5,25 +5,22 @@
 // make sure webfont brings in the icons and css files.
 import "@bentley/icons-generic-webfont/dist/bentley-icons-generic-webfont.css";
 import "./AppComponent.css";
-
 import * as React from "react";
 import { Provider } from "react-redux";
-
-import { Config, Id64, OpenMode } from "@bentley/bentleyjs-core";
+import { Config, GuidString, Id64 } from "@bentley/bentleyjs-core";
 import { IModelHubClient, VersionQuery } from "@bentley/imodelhub-client";
-import { IModelVersion } from "@bentley/imodeljs-common";
+import { BriefcaseIdValue } from "@bentley/imodeljs-backend";
```

Comment:
@kabentley I did a bunch of profiling, and it looks like this import is causing webpack to pull in all of imodeljs-backend - that's probably the cause of most of the webpack issues...

Should this enum be moved to imodeljs-common?

Reply:
Yeah, I have a webpack plugin to flag backend imports from frontend code (and vice versa), but we must not have added that to our create-react-app fork... I'll look into whether we can add that back in there...

I'm also working on some sass-loader changes that should significantly speedup the build time here, but that will at a minimum require us to publish a new react-scripts version.  Hopefully we can try that here by the end of the day.

Reply:
Just pushed my build performance fixes.  By disabling ESLint, Babel (for node_modules), and Terser, it looks like we've got the CI build time down from [5m 25s](https://github.com/imodeljs/desktop-starter/pull/47/checks?check_run_id=2007711063) to [2m 0s](https://github.com/imodeljs/desktop-starter/pull/47/checks?check_run_id=2032371495).